NuRadioMC/SignalProp/directRayTracing.py

```diff
@@ -33,6 +33,9 @@ def set_start_and_end_point(self, x1, x2):
         self._x1 = x1
         self._x2 = x2
 
+    def set_shower_axis(self,shower_axis=None):
+        self._shower_axis = shower_axis
```

Comment:
In many cases it is already normallised I think. I don't now if it would be useful since it is only used for directional purposes. The line can be added though.